MAINTAINERS.md

```diff
@@ -5,9 +5,20 @@ The current Maintainers Group for the Jaeger Project consists of:
 | [@albertteoh](https://github.com/albertteoh) | PackSmith | ALL | 
 | [@jkowall](https://github.com/jkowall) | Aiven | ALL |
 | [@joe-elliott](https://github.com/joe-elliott) | Grafana Labs | ALL |
+| [@mahadzaryab1](https://github.com/mahadzaryab1) | ? | ALL |
```

Comment:
@mahadzaryab1 what should we put for your Employer?

Reply:
Bloomberg